src/main/java/io/pravega/perf/PravegaPerfTest.java

```diff
@@ -80,6 +80,7 @@ public static void main(String[] args) {
                         "if -1, get the maximum throughput");
         options.addOption("writecsv", true, "CSV file to record write latencies");
         options.addOption("readcsv", true, "CSV file to record read latencies");
+        options.addOption("enableConnectionPooling", true, "enable connection pooling");
```

Comment:
you need update the readme file also

Reply:
README updated. Commit rebased onto latest master.